src/Makefile.common

```diff
@@ -48,8 +48,8 @@ $(top_builddir)/src/include/version.hh: $(top_builddir)/Makefile.conf $(REALTOPD
 	echo "#define	VERDATE \"$(RELEASE_DATE)\"" >> $@; \
 	echo "#endif /* VERSION_H */" >> $@
 
-$(top_builddir)/src/base/bios/bios_offsets.hh:
-	$(MAKE) -C $(top_builddir)/src/base/bios bios_offsets.hh
+$(top_builddir)/src/base/bios/bios_offsets.hh $(top_builddir)/src/base/bios/bios_data.xxd:
+	$(MAKE) -C $(top_builddir)/src/base/bios bios_offsets.hh bios_data.xxd
```

Comment:
I think you don't need xxd here because
`bios_offsets.hh` should instead depend on it.